shoov/modules/custom/shoov_ci_build/shoov_ci_build.features.field_base.inc

```diff
@@ -79,6 +79,33 @@ function shoov_ci_build_field_default_field_bases() {
     'type' => 'list_boolean',
   );
 
+  // Exported field_base: 'field_ci_build_incident_status'
+  $field_bases['field_ci_build_incident_status'] = array(
+    'active' => 1,
+    'cardinality' => 1,
+    'deleted' => 0,
+    'entity_types' => array(),
+    'field_name' => 'field_ci_build_incident_status',
+    'indexes' => array(
+      'value' => array(
+        0 => 'value',
+      ),
+    ),
+    'locked' => 0,
+    'module' => 'list',
+    'settings' => array(
+      'allowed_values' => array(
+        0 => 'new',
+        1 => 'done',
+        2 => 'unconfirmed error',
+        3 => 'error',
+      ),
+      'allowed_values_function' => '',
+    ),
+    'translatable' => 0,
+    'type' => 'list_integer',
```

Comment:
lets use a non-required a list_text with the options:

unconfirmed error
error

We then can use NULL as indicate that it's ok.